moped-editor/src/utils/mapHelpers.js

```diff
@@ -616,11 +616,10 @@ export function useFeatureCollectionToFitBounds(
      */
     const fitViewportToBounds = viewport => {
       if (featureCollection.features.length === 0) return viewport;
-      debugger;
       const featureViewport = new WebMercatorViewport({
         viewport,
-        width: currentMap?._width ?? window.innerWidth,
-        height: currentMap?._height ?? window.innerHeight,
+        width: currentMap?._width ?? (window.innerWidth * 0.80),
```

Comment:
@sergiogcx this looks good on my end. confirming that project maps are rendering now. I agree there's a better way to fix this but this gets it done for now.